src/Tools/ExternalAccess/Razor/Cohost/RazorRequestExecutionQueueProvider.cs

```diff
@@ -16,7 +16,7 @@ namespace Microsoft.CodeAnalysis.ExternalAccess.Razor.Cohost;
 [method: Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
 internal sealed class RazorRequestExecutionQueueProvider() : IRequestExecutionQueueProvider<RequestContext>
 {
-    public IRequestExecutionQueue<RequestContext> CreateRequestExecutionQueue(AbstractLanguageServer<RequestContext> languageServer, ILspLogger logger, IHandlerProvider handlerProvider)
+    public IRequestExecutionQueue<RequestContext> CreateRequestExecutionQueue(AbstractLanguageServer<RequestContext> languageServer, ILspLogger logger, AbstractHandlerProvider handlerProvider)
     {
         var queue = new RoslynRequestExecutionQueue(languageServer, logger, handlerProvider);
```

Comment:
Do we need our own Queue now too, to override `GetLanguage` so that our handlers are found, now trhat they're being tagged with `RazorMethod`?